api/MediaList.json

```diff
@@ -238,6 +238,45 @@
             "deprecated": false
           }
         }
+      },
+      "toString": {
+        "__compat": {
+          "spec_url": "https://w3c.github.io/csswg-drafts/cssom/#MediaList-stringification-behavior",
```

Comment:
So the spec link here points to: `stringifier attribute [LegacyNullToEmptyString] CSSOMString mediaText;`. You already documented `mediatext`. 

Does the existence of `stringifier` on any attribute force the existence of a `toString` method? 

If so I'm OK with this, though it has similar issues to "what do I put down in BCD/docs if I get this odd symbol" - see https://github.com/mdn/browser-compat-data/pull/19094.

We should update the data guidelines and comment in https://github.com/mdn/browser-compat-data/issues/6367#issuecomment-863126606 too. Thoughts?

Reply:
Also see https://github.com/mdn/browser-compat-data/pull/19091/files#r1134750585 which words the problem slightly differently - "if `stringifer` implies the existence of `toString`, but this doesn't differ in any way from the value that it is applied to, then what is the point in separately indicating toString exists?

I don't mind, I just want to know that there is a clear pattern we adopt in all cases, and this is documented.

Reply:
> Does the existence of `stringifier` on any attribute force the existence of a `toString` method?

That's correct, the `stringifier` attribute creates a `toString()` method that is essentially a getter for that attribute!

Reply:
So what's the point? You can get the same thing from the associated value - does it help to have this as well? 

I think it does; what I'm really asking for is a note in the schema doc somewhere that says you should add toString whenever you see the `stringifier` - that way the rule is clear and documented.

Reply:
> what I'm really asking for is a note in the schema doc somewhere that says you should add toString whenever you see the `stringifier` - that way the rule is clear and documented.

Adding notes like this to the guidelines sounds good to me!  https://github.com/mdn/browser-compat-data/pull/19121

Reply:
In regards to the data itself, how does it look to you @hamishwillee?